### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GauravSarkar**
Added the version of hugo.
corrected in response to issue #8
Signed-off-by: GauravSarkar <60042820+GauravSarkar@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 This repository contains a boilerplate static site generator setup for creating CNCF documentation projects. We strongly recommend using this setup (it helps us help you and your project!), but none of the technologies in the stack are strictly required.
 
 The starter uses the following:
-* **[Hugo](https://gohugo.io/)** as a static site generator
+* **[Hugo](This requires Hugo 9.73+ extended version)(https://gohugo.io/)** as a static site generator
```
Comment (nate-double-u) [inline README.md:6]: For markdown links, the whole text to be linked needs to be within the `[]` brakets

```suggestion
* **[Hugo (This requires Hugo 9.73+ extended version)](https://gohugo.io/)** as a static site generator
```

**Comment by nate-double-u** [comment]
Hi @GauravSarkar, thanks for looking into this!

I saw your [comment in the original issue](https://github.com/cncf/hugo-netlify-starter/issues/8#issuecomment-717913328), but would like to respond to it here as the DCO issues are a part of this PR process.

I'm not sure if there is a sign-off method (like a checkbox) in the GitHub UI (like there are in some other git GUIs). Because the sign-off is just a line of text in the commit, it is possible to sign a commit correctly without using the command line. 
However, I do recommend getting familiar with the command line as it makes some of these issues easier to deal with.

Looking at the details of the [DCO check](https://github.com/cncf/hugo-netlify-starter/pull/49/checks?check_run_id=1320598115), it looks like there is a slight discrepancy between what has been provided, and what github is looking for in a sign-off (I wonder if there is a discrepancy between your name in GitHub, and your local .gitconfig name?).

> Commit sha: 5428c91, Author: Gaurav Sarkar, Committer: GitHub; Expected "Gaurav Sarkar 60042820+GauravSarkar@users.noreply.github.com", but got "GauravSarkar 60042820+GauravSarkar@users.noreply.github.com".

**Review by nate-double-u — CHANGES_REQUESTED** [review]
Please update link to gohugo.io

**Comment by GauravSarkar** [comment]
Thanks I have finally sorted out the DCO error and to avoid confusions I have opened a fresh PR #53  with the required changes.

**Comment by nate-double-u** [comment]
Great, I'll take a look at that PR and close this one.

**Pull request closed without merging**